08LessonEight/in-class-project/homework.html

```diff
@@ -13,28 +13,58 @@
 
     <!--begin main section-->
     <main>
-        <section>
-            <img src="img/keyboard.jpg">
+        <section id="keyboard">
+            <img id="keyboard_img" src="img/keyboard.jpg">
             <h3>BEEP BEEP BOOP</h3>
             <p>blah blah blah type look at me type I am typing things in my blog please pay no attention to the ugly yellow watch</p>
         </section>
-        <aside>
             <div class="three">
                 <div id="phones">
+                  <img src="img/laptop.jpg">
                     <h3>Weird Headphones</h3>
                     <p>these are my weird ugly headphones</p>
                 </div>
                 <div id="laptop">
+                  <img src="img/desk.jpg">
                     <h3>Dumb Div</h3>
                     <p>not really sure what to call this one</p>
                 </div>
                 <div id="window">
+                  <img src="img/window.jpg">
                     <h3>I'm blind</h3>
                     <p>this light is really bright and now I'm blind</p>
                 </div>
             </div>
-        </aside>
     </main>
+<clearfix class="clearfix"></clearfix>
+    <footer>
+      <div class="lists">
+        <div id="profession">
+        <h3>Why Profession?</h3>
+        <ul>
+          <li>bullshit</li>
+          <li>bullshit</li>
```

Comment:
this may be bullshit. BUT IT STILL NEEDS TO BE COMPLETED. :dancer:  needs to be spaced and have borders